lib/gui/components/safe-webview/safe-webview.js

```diff
@@ -54,63 +57,64 @@ class SafeWebview extends react.PureComponent {
    * @returns {react.Element}
    */
   render() {
-    if (this.state.shouldLoad) {
-      const url = new URL(this.props.src);
-
-      // We set the 'etcher-version' GET parameter here.
-      url.searchParams.set('etcher-version', packageJSON.version);
-
-      return react.createElement('webview', {
-        ref: 'webview',
-        src: url.href
-      }, []);
-    }
-
-    // We have to return null explicitly, undefined is an error in React.
-    return null;
+    return react.createElement('webview', {
+      ref: 'webview',
+      src: this.makeURL().href,
+      style: {
+        flex: this.state.shouldShow ? null : '0 1',
+        width: this.state.shouldShow ? null : '0',
+        height: this.state.shouldShow ? null : '0'
+      }
+    }, []);
   }
 
   /**
-   * @summary Add the Webview events if there is an element
+   * @summary Add the Webview events
    */
   componentDidMount() {
 
-    // There is no element to add events to if 'shouldLoad' is false.
-    if (this.state.shouldLoad) {
+    // Events React is unaware of have to be handled manually
```

Comment:
minor nit, but could you change this to "Events that React is ...", otherwise "Events React" might sound like another framework ;)